### PR TITLE
Build 9.9.0

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Channel: ❄️ 
Upstream: https://github.com/VowpalWabbit/vowpal_wabbit/tree/9.9.0

Changes from conda-forge:
* Use our own sse2neon, ensmallen and armadillo packages instead of the bundled packages.
* Make build verbose (to help debugging and code review)

Depends on:
* https://github.com/AnacondaRecipes/armadillo-feedstock/pull/1
* https://github.com/AnacondaRecipes/sse2neon-feedstock/pull/1
* https://github.com/AnacondaRecipes/ensmallen-feedstock/pull/1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,43 +11,53 @@ source:
...
-    - spdlog
-    - rapidjson
-    - fmt
-    - eigen
+    - libboost {{ boost }}
+    - py-boost {{ boost }}
+    - zlib {{ zlib }}
+    - spdlog >=1.9.2
+    - rapidjson  >=1.1.0
+    - fmt >=8.1.1
```
Comment (cbouss) [inline recipe/meta.yaml:48]:
```suggestion
    - fmt 9.1.0 # >=8.1.1 fmt has a run_exports on the major version
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,43 +11,53 @@ source:
...
-    - zlib
-    - spdlog
-    - rapidjson
-    - fmt
-    - eigen
+    - libboost {{ boost }}
+    - py-boost {{ boost }}
+    - zlib {{ zlib }}
+    - spdlog >=1.9.2
+    - rapidjson  >=1.1.0
```
Comment (cbouss) [inline recipe/meta.yaml:47]:
```suggestion
    - rapidjson  1.1.0 # >=1.1.0
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,43 +11,53 @@ source:
...
     - armadillo
     - ensmallen
     - sse2neon  # [arm64 or aarch64 or ppc64le]
+    - setuptools
+    - wheel
   run:
     - python
     - zlib
-    - spdlog
     - rapidjson
```
Comment (cbouss) [inline recipe/meta.yaml:58]:
```suggestion
    - - {{ pin_compatible('rapidjson') }} # no run_exports on rapidjson
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -57,7 +67,7 @@ test:
     - vowpalwabbit
   commands:
     - pip check
-    - python -m pytest python/tests/ -k "not test_ccb_and_automl"  # [python_impl == "cpython" and linux64]
+    - python -m pytest -v python/tests/ -k "not test_ccb_and_automl"
```
Comment (cbouss) [inline recipe/meta.yaml:70]:
```suggestion
    # test_ccb_and_automl requires package vw_executor
    - python -m pytest -v python/tests/ -k "not test_ccb_and_automl"
```

**Review by cbouss — APPROVED** [review]

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Comment by M-Waszkiewicz-Anaconda** [comment]
@boldorider4 have you tried on windows 
https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Building#windows-specific-info ?

**Pull request merged**